api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -840,12 +841,34 @@ def get_managed_venues(offerer_id: int) -> utils.BackofficeResponse:
 @repository.atomic()
 def get_offerer_addresses(offerer_id: int) -> utils.BackofficeResponse:
     offerer_addresses = (
-        offerers_models.OffererAddress.query.filter_by(offererId=offerer_id)
-        .options(
-            sa.orm.load_only(offerers_models.OffererAddress.label),
-            sa.orm.joinedload(offerers_models.OffererAddress.address),
+        db.session.query(
+            geography_models.Address.id,
+            geography_models.Address.street,
+            geography_models.Address.postalCode,
+            geography_models.Address.city,
+            geography_models.Address.banId,
+            geography_models.Address.latitude,
+            geography_models.Address.longitude,
+            sa.func.array_agg(
+                sa.func.coalesce(
+                    offerers_models.OffererAddress.label, offerers_models.Venue.name, offerers_models.Venue.publicName
```

Comment:
Je crois qu'il faut d'abord mettre le publicName - qui est optionel - avant de venue.name (cf [api/src/pcapi/core/offerers/models.py](https://github.com/pass-culture/pass-culture-main/blob/7eb52ff50053ff536c191c81d39028520b44e303/api/src/pcapi/core/offerers/models.py#L664))
```suggestion
                    offerers_models.OffererAddress.label, offerers_models.Venue.publicName, offerers_models.Venue.name
```

Reply:
tout à fait, c'est fix